test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperAttributesTest.cs

```diff
@@ -0,0 +1,717 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNet.Razor.Runtime.Test;
+using Xunit;
+
+namespace Microsoft.AspNet.Razor.Runtime.TagHelpers
+{
+    public class TagHelperAttributesTest
+    {
+        public static TheoryData IntIndexerGetData
```

Comment:
optionally could also inherit from `ReadOnlyTagHelperAttributesTest` and give that class `protected virtual ReadOnlyTagHelperAttributes<IReadOnlyTagHelperAttribute> CreateInstance(...)` and `... CreateTestInstance(...)` methods.